test/unittest/dynamic_types/DynamicTypes_4_2_Tests.cpp

```diff
@@ -162,7 +162,7 @@ TEST_F(DynamicTypes_4_2_Tests, New_Union_Discriminators)
 
     ASSERT_TRUE(sizeof(struct_test.int8Union()._d()) == 1);
     ASSERT_TRUE(sizeof(struct_test.octetUnion()._d()) == 1);
-    ASSERT_TRUE(sizeof(struct_test.charUnion()._d()) == sizeof(wchar_t));
+    //ASSERT_TRUE(sizeof(struct_test.charUnion()._d()) == sizeof(wchar_t));
```

Comment:
I suggest leaving a TODO to remember to uncomment after introducing the Dynamic Types refactor.

Reply:
The TODO is located in `new_features_4_2.idl` file